docs/extras/blocking_updates.md

```diff
@@ -1,12 +1,35 @@
 # Blocking Updates
 
-You can add a custom DNS to your WiFi connection that will block all communication with Nintendo's servers. This will stop any updates and reporting to Nintendo, but it will prevent use of the eShop and online games.
+This page will help you set up a method to block Nintendo from reaching your switch. This will stop any updates and reporting to Nintendo, but it will prevent use of the eShop and online games.
 
-We will be using [90DNS](https://gitlab.com/a/90dns), a community-run custom DNS server. If you prefer, you can run your own DNS server following the instructions on the [GitLab repository.](https://gitlab.com/a/90dns/blob/master/SELFHOST.md)
+### Instructions (AMS DNS redirection)
+You can configure atmosphere to automatically redirect any requests directed to Nintendo to nothing instead. Documentation about this feature can be [found here](https://github.com/Atmosphere-NX/Atmosphere/blob/master/docs/features/dns_mitm.md). The section below will help you set up dns redirection on your emummc. Note that this will only apply when you are using cfw.
+
```

Comment:
Small thing but it would probably look better if capitalization was consistent on this page (Nintendo, DNS, eShop, SD, etc).